### PR TITLE
I can't take it any more. Sort the GEP list.

### PR DESCRIPTION
**Pull request by kflynn**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

I can't handle the randomly-sorted GEPs in the sidebar any more. :joy: Let's keep them numerically sorted, please? :slightly_smiling_face:

**Does this PR introduce a user-facing change?**:
-->
```release-note
NONE
```

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -93,32 +93,32 @@ nav:
...
-        - geps/gep-1323.md
-        - geps/gep-922.md
-        - geps/gep-917.md
-        - geps/gep-851.md
-        - geps/gep-820.md
-        - geps/gep-746.md
-        - geps/gep-726.md
-        - geps/gep-724.md
-        - geps/gep-718.md
         - geps/gep-709.md
```
Comment (robscott) [inline mkdocs.yml:111]: One nitpick: This was intentionally done in descending order so the most recent GEPs would be at the top of the list. I don't have a strong opinion here, and clearly other parts of this nav have it in ascending order so 🤷.

**Review by robscott — COMMENTED** [review]
Thanks @kflynn! Don't care too much whether this is ascending or descending order, but completely agree that this should be consistent.

/approve

**Comment by robscott** [comment]
@kflynn do you have time to follow up on this one?

File: mkdocs.yml (hunk 1 above)
Comment (kflynn) [reply mkdocs.yml:112]: I (eventually) decided that I'd prefer having them all ascending. Not a strong opinion here either, really -- anyone else feel strongly?

**Comment by kflynn** [comment]
Rebased! Sorry for the delay.

File: mkdocs.yml (hunk 1 above)
Comment (youngnick) [reply mkdocs.yml:112]: I think consistency is better here. It's not a big enough deal to worry about too much - we can always change it later.

**Pull request merged**